### PR TITLE
Issue 1 create an entry Jimmy Tran

### PR DESCRIPTION
**Pull request by JimmyTran1206**
Animation

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  font-size: 1rem;
+}
+
+h2.page-title {
+  font-size: 1.5rem;
+}
+
+img.entry-image {
+  width: 100%;
+  height: 15rem;
```
Comment (shawnkost) [inline css/styles.css]: We don't want to give this a `width` & a `height`. We only want to give this a `width`. We want the browser to automatically adjust the `height` for us, so it'll maintain the aspect ratio. By setting a `height`, you're distorting the image.

File: index.html [hunk 2]
```diff
@@ -18,11 +18,75 @@
...
+              </div>
+            </div>
+
+            <!-- PhotoURL -->
+            <div class="label-input-div">
+              <div class="label-div">
+                <label for="photo-url">Photo URL</label>
+              </div>
+              <div class="input-div">
+                <input type="url" name="photoURL" id="photoURL" class="input-photo-url" required="required">
```
Comment (shawnkost) [inline index.html]: The `id` for this `input` should be `photo-url` to match the `for` attribute of your `label` on line 56

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+function formSave(event) {
+  event.preventDefault();
+  const formData = {};
+  formData[$form.elements.title.name] = $form.elements.title.value;
+  formData[$form.elements.photoURL.name] = $form.elements.photoURL.value;
+  formData[$form.elements.notes.name] = $form.elements.notes.value;
+  formData.entryId = data.nextEntryId;
+  data.nextEntryId++;
+  data.entries.push(formData);
```
Comment (shawnkost) [inline js/main.js]: `formData` should be getting added to the beginning of the array

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Jimmy! Some changes requested below! As well as a couple other changes here that I didn't leave a comment on.

1. The text inside of your header should be aligned with the `img` & "notes" text.
2. Same thing with the "New Entry" text. This text should be aligned with the image below it.

Basically all of your text should be aligned and start at the same spot.

Let me know if you have any questions!

File: css/styles.css (hunk 1 above)
Comment (JimmyTran1206) [reply css/styles.css]: Hi Shawn, thank you for your comment. I address this issue by removing the `height` property. However, as I saw that in the demo of issue #1, the image became a rectangle in mobile view and a square in desktop view with the center of the image undistorted. Therefore, in mobile view, I set a height and set object-fit: cover so the image is cropped to fit the view, but it stays undistorted.

File: index.html (hunk 2 above)
Comment (JimmyTran1206) [reply index.html]: I address this issue by changing the `for` attribute of the label to photoURL so I do not need to change the JavaScript query selection.

File: js/main.js (hunk 3 above)
Comment (JimmyTran1206) [reply js/main.js]: Thank you for this comment. I address this issue by using array.unshift() method instead of array.push().

**Comment by JimmyTran1206** [comment]
> Looking good so far Jimmy! Some changes requested below! As well as a couple other changes here that I didn't leave a comment on.
> 
> 1. The text inside of your header should be aligned with the `img` & "notes" text.
> 2. Same thing with the "New Entry" text. This text should be aligned with the image below it.
> 
> Basically all of your text should be aligned and start at the same spot.
> 
> Let me know if you have any questions!

I address this issue by putting the header 'Code Journal' and the form title 'New Entry' into `<div class="column-full">`. I have pushed the code again.

**Comment by JimmyTran1206** [comment]
image
Animation
Updated result with screenshots

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**